src/status_im/contexts/shell/jump_to/components/switcher_cards/view.cljs

```diff
@@ -294,32 +294,35 @@
 
 (defn card
   [{:keys [type] :as data}]
-  (cond
-    (= type shell.constants/empty-card) ; Placeholder
-    [empty-card]
+  (let [screen-width (:width (rn/get-window))
+        ;; 2 column, 20px horizontal margin, 15px gap
+        card-size    (rn/use-memo #(/ (- screen-width 55) 2) [screen-width])]
```

Comment:
to me here use of use-memo don't look much beneficial.
- This will be cached for all switcher cards
- This is not a big calculation, and switcher cards don't change that often.